components/Tabs/BUILD

```diff
@@ -80,6 +80,17 @@ mdc_extension_objc_library(
     ],
 )
 
+mdc_extension_objc_library(
+    name = "Theming",
+    deps = [
+        ":Tabs",
+        "//components/schemes/Container",
+        "//components/schemes/Color",
+        "//components/schemes/Typography",
+    ],
+
```

Comment:
Nit: Please run `buildifier` or remove this empty line.

Reply:
Addressed in 4e8cb3cc3ec6eb9a0b5536dac519b6b866791415